inc/spbc-scanner.php

```diff
@@ -1108,19 +1109,21 @@ function spbc_get_sql_where_addiction_for_table_of_category($category)
  * @param string $category Category of files category which needs to be searched for
  * @return array Array of IDs
  */
-function spbc_scanner_get_files_by_category($category)
+function spbc_scanner_get_files_by_category($category, $count = false)
```

Comment:
Please, update doc-block with new param and rerun tests

Reply:
done https://github.com/CleanTalk/security-malware-firewall/pull/444/commits/dc6097193ecabd4c35372180045a6d43a4a97f04